### ISSUE TITLE
NVDA and Avast

### ISSUE THREAD

**Issue by cocodebordeaux** [nvaccess/nvda#5633]
Hello, 
it seems that NVDA cannot read window contents in Avast 2016 antivirus configuration dialogs. 
When pressing tab to access the different parameters, they are not detected by NVDA.
As JAWS can do it, it looks like a NVDA problem.
. 
 It should therefore solve this little bug. 
 Thank you.

**Comment by leonardder** [comment]
@cocodebordeaux: Can you still reproduce this issue?

**Comment by ehollig** [comment]
Duplicate of #4314 3241

**Comment by ehollig** [comment]
Closing as duplicate of #3241. If this is still happening, please create a new issue and provide exact detailed steps to reproduce, including what you did, what you expected to happen and what happened instead.
